src/accordionpane/example/index.ts

```diff
@@ -0,0 +1,61 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { Set } from '@dojo/shim/Set';
+import { w, v } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties, TypedTargetEvent } from '@dojo/widget-core/interfaces';
+
+import AccordionPane from '../../accordionpane/AccordionPane';
+import TitlePane from '../../titlepane/TitlePane';
+
+import dojoTheme from '../../themes/dojo/theme';
+
+export class App extends WidgetBase<WidgetProperties> {
+	private _openKeys = new Set<string>();
+	private _theme: {};
+
+	themeChange(event: TypedTargetEvent<HTMLInputElement>) {
+		this._theme = event.target.checked ? dojoTheme : {};
+		this.invalidate();
+	}
+
+	render() {
+		return v('div', { styles: { maxWidth: '350px' } }, [
+			v('h2', [ 'AccordionPane Examples' ]),
+			v('label', [
+				'Use Dojo Theme ',
+				v('input', {
+					type: 'checkbox',
+					onchange: this.themeChange
+				})
+			]),
+			v('div', { id: 'pane' }, [
+				w(AccordionPane, {
+					onRequestOpen: (key: string) => {
+						this._openKeys.add(key);
+						this.invalidate();
+					},
+					onRequestClose: (key: string) => {
+						this._openKeys.delete(key);
+						this.invalidate();
+					},
+					openKeys: Array.from(this._openKeys),
+					theme: this._theme
+				}, [
+					w(TitlePane, {
+						title: 'Pane 1',
+						key: 'foo'
+					}, [ 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque sodales ante sed massa finibus, at euismod ex molestie. Donec sagittis ligula at lorem blandit imperdiet. Aenean sapien justo, blandit at aliquet a, tincidunt ac nulla. Donec quis dapibus est. Donec id massa eu nisl cursus ornare quis sit amet velit.' ]),
+					w(TitlePane, {
+						title: 'Pane 2',
+						key: 'bar'
+					}, [ 'Ut non lectus vitae eros hendrerit pellentesque. In rhoncus ut lectus id tempus. Cras eget mauris scelerisque, condimentum ante sed, vehicula tellus. Donec congue ligula felis, a porta felis aliquet nec. Nulla mi lorem, efficitur nec lectus vehicula, vehicula varius eros.' ])
+				])
+			])
+		]);
```

Comment:
Since we're saying `exclusive` is the responsibility of the parent, I think it'd be nice to include an example of it here.

Reply:
Added.